decisions/30-release-workflow.md

```diff
@@ -0,0 +1,37 @@
+# Use Release Process Based on GitFlow
+
+Discussion: https://github.com/ESIPFed/science-on-schema.org/issues/30
+
+## Status ##
+_Proposed_
+
+## Decision ##
+
+We will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model to track proposed changes, their implementation, and the associated Architectural Decision Record.
+
+## Context  -- What is the issue that we're seeing that is motivating this decision or change.
+Because people will review our guidance documents online, it will be helpful to be explicit about versioning and releases of the guidance and other documents.  In particular, in GitHub, users will normally see the `master` branch, which therefore should reflect the current stable release of the documentation (rather thanb confusing people with in-progress proposed changes that are not yet released).
+
+Consequently, we will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model in which the `master` branch always reflects the current stable release, a `develop` branch is used for merging finished proposals being prepared for release, and `feature` branches are used for creating changes to implement specific proposals that are reflected in an Architectural Decision Record.  For changes that do not require a formal decision via and ADR, such as spelling corrections, grammatical rewording, etc., maintainers can commit changes directly to the `develop` branch, and other contributors can do a pull request directly against the `develop` branch.  The use of feature branches is really focused on managing proposals that need discussion, review, and a dec ision through an ADR. Maintainers will make judgement calls on whether an ADR is needed, and might convert contributed pull requests to a feature branch if they determine that an ADR is needed.
```

Comment:
... and `feature` branches ~~are used for creating changes to~~ **for** implement**ing** specific ...

... discussion, review, and a ~~dec ision~~**decision** through an ADR. ...